warehouse-management-service/internal/handler/handler_test.go

```diff
@@ -887,10 +915,14 @@ func TestUpdateShelfBlockRequestError(t *testing.T) {
 				StorageType: "",
 				WarehouseId: "",
 			},
-			want:           api.ShelfBlockResponse{Error: "Invalid input: storageType cannot be empty, warehouseId cannot be empty"},
+			want: []string{
+				"StorageType: zero value",
+				"WarehouseId: zero value",
+			},
 			wantStatusCode: http.StatusBadRequest,
 		},
 	}
+
 	for _, test := range tests {
```

Comment:
Not a fan of the table driven test style personally (for poor stack traces + readability overheads). But, it's very ecosystem appropriate 👍